package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.0.1",
   "description": "AeroGear Sync Server UI",
   "main": "index.js",
-  "license": "MIT",
+  "license": "Apache-2.0",
+  "repository": "https://github.com/aerogear/data-sync-ui",
   "scripts": {
-    "server": "webpack --mode development && nodemon --watch ui index.js",
+    "lint": "eslint server test ui webpack.config.js --fix",
+    "server": "webpack --mode development && nodemon --watch ui --exec babel-node server/index.js",
     "build": "webpack --mode production",
```

Comment:
don't we have to take `babel-node` into account here? In the docker container we are not using babel, we need to provide something that can be started with pure node.

Reply:
Yes that's another lost during rebase, it's fixed now.